WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -198,15 +198,18 @@ decisions = {
 			age = 16
 			scaled_wealth = 0.5
 			prisoner = no
-			religion_group = christian
-			NOT = { has_character_flag = VIET_celtic_high_cross }
 			OR = {
-				culture = saxon
+				religion = insular
+				religion = pelagianism
 				AND = {
-					culture_group = celtic
-					NOT = { culture = breton }
+					religion_group = christian
+					OR = {
+						culture = saxon
+						culture = old_saxon
+					}
```

Comment:
When, where, how, and for what purpose?

I added the provision that Christian Saxons can always erect High Crosses, like you requested. If non-Christian Saxons erected monuments, too, then that should either be its own Decision, or you should use the Raise a Runestone Decision. I reccomend the latter.